CONTRIBUTING.md

```diff
@@ -27,6 +27,10 @@ to manage the issues. Open an issue directly in this issue tracker.
 If you believe you found a bug, and it's likely possible, please indicate a way to reproduce it, what you are seeing and what you would expect to see.
 Don't forget to indicate your Narayana, Java and Maven version. 
 
+## Making open source more inclusive
+
+Red Hat is committed to eradicating problematic language in our code, documentation, web properties, and interactions (https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language). Although we have no plans to retrospectively update our materials we do ask that all contributors be cognizant of the language they use in all communications related to narayana.
```

Comment:
Please can I request capitalizing of "Narayana"

Reply:
I would also add that the "we have" in this sentence presumably refers to the project, e.g. "Although the Narayana project has"

Reply:
Done, I also slightly rejigged the sentence.

Reply:
And I have just used your suggested amendment.

Reply:
which had the knock on effect of requiring that I replace `our` with `its` so I pushed that change